docs/api/en/core/InterleavedBufferAttribute.html

```diff
@@ -95,7 +95,8 @@ <h3>[method:null setXYZ]( [param:Integer index], [param:Float x], [param:Float y
 		<h3>[method:null setXYZW]( [param:Integer index], [param:Float x], [param:Float y], [param:Float z], [param:Float w] ) </h3>
 		<p>Sets the x, y, z and w components of the item at the given index.</p>
 
-
+		<h3>[method:null dispose]()</h3>
+		<p>Disposes the buffer attribute from memory directly.Also you can dispose the buffer attribute through [page:BufferGeometry.dispose BufferGeometry.dispose] implicitly.</p>
```

Comment:
I don't see how disposing an InterleavedBufferAttribute would work, since it would share an underlying InterleavedBuffer with other attributes. If we add this method, I think that invoking it on an interleaved attribute should throw an error.

Reply:
`InterleavedBufferAttribute` is not derived from `BufferAttribute`. So I think a note in the docs is sufficient.